README.md

```diff
@@ -1,136 +1,96 @@
 
-# **D**jango **R**epository for **ME**vo**L**ib **W**eb-based **G**raphical **U**ser **I**nterface
+# Django Repository for MEvoLib Web-based Graphical User Interface
 
 First of all, MEvoLib is a library of freely available Python tools for molecular evolution.
+
 This project has been developed based on the use of that library, keeping in mind to provide users:
-    - An easier and more visual way to interact with MEvoLib.
-    - A more controlled process, not only because of custom stages that will be run within the workflow (Fetch, Cluster, Align and Inference,
-    for now), but also by providing all the available parameters for each stage in a single sight, listing the possibilities for each one.
-    - A simplier way to get the results, as they will be stored in a ZIP that will be downloaded in the local machine after each requested
-    process has finished. Even if it fails, a filtered log message will be displayed, so that the user has a clue of what went wrong. 
+- An easier and more visual way to interact with MEvoLib.
+- A more controlled process, not only because of custom stages that will be run within the workflow (Fetch, Cluster, Align and Inference, for now), but also by providing all the available parameters for each stage in a single sight, listing the possibilities for each one.
+- A simpler way to get the results, as they will be stored in a ZIP that will be downloaded in the local machine after each requested process has finished. Even if it fails, a filtered log message will be displayed, so that the user has a clue of what went wrong.
 
-# **L**icense **T**erms
+## License terms
 
 This Django package is open source software made available under the Apache 2.0 License terms. Please see the LICENSE file for further details.
 
-# **I**nstallation **P**rocess
+## Installation
 
-- The first step is to clone this library. You can do it by typing this line in your console:
+The first step is to clone this library. You can do it by typing this line in your console:
 ```bash
 git clone https://github.com/nachorequeno/mevolib-ui.git
 ```
 
-- After that, you have to enter to the folder:
+Next, you have to install all of the dependencies:
 ```bash
-cd mevolib-ui
+pip install mevolib-ui/MEvoLibGUI/.
 ```
 
-- Then, you have to install all of the dependencies:
+After that, generate the secret key and save it in a file named `.env`. You can do it by running the following command:
 ```bash
-pip install -e.
+python manage.py shell -c "from django.core.management.utils import get_random_secret_key; print(f'SECRET_KEY = {get_random_secret_key()}')" > .env
 ```
 
-- When you are done with this, the next step make sure you have installed python-dotenv to store the secret key provided by Django.
-If you have not it, you can type:
+Now clone `mevolib-workflow` repository too:
 ```bash
-pip install python-dotenv
```

Comment:
Why did you deleted this? Is it not necessary?

In your settings.py file, you have these lines: 
**_from dotenv import load_dotenv
load_dotenv()
SECRET_KEY = os.environ.get("SECRET_KEY")_**

So, I am unsure whether it may create some conflicts...

Reply:
It exists as a dependency in `pyproject.toml` so `pip` installs it already.